addons/account/wizard/wizard_tax_adjustments.py

```diff
@@ -20,11 +23,11 @@ def _get_default_journal(self):
                                         domain="[('deprecated', '=', False), ('is_off_balance', '=', False)]")
     amount = fields.Monetary(currency_field='company_currency_id', required=True)
     adjustment_type = fields.Selection([('debit', 'Applied on debit journal item'), ('credit', 'Applied on credit journal item')], string="Adjustment Type", required=True)
-    tax_report_line_id = fields.Many2one(string="Report Line", comodel_name='account.tax.report.line', required=True, help="The report line to make an adjustment for.")
+    tax_report_line_id = fields.Many2one(string="Report Line", comodel_name='account.tax.report.line', required=True, help="The report line to make an adjustment for.",
+                                         domain=_domain_tax_report)
```

Comment:
It is ok for the stable version (because we can't add computed fields in stable...), but something like this in master should be better
```suggestion
    tax_report_line_id = fields.Many2one(
        string="Report Line", 
        comodel_name='account.tax.report.line', 
        required=True,
        domain="[('tag_name', '!=', None), ('report_id.country_id', '=', 'company_id')]",
        help="The report line to make an adjustment for.",
    )
```

With a computed field `company_id` related on the journal's company